furusiya/model/keys/update_manager.py

```diff
@@ -1,4 +1,5 @@
 from model.config import config
+from game import Game
```

Comment:
Why is this imported here? I thought UpdateManager was handed an instance of Game. What's this for?

Reply:
Because I don't like passing `game` around some places and using `Game.instance` other places. I didn't get around to removing the import after I backed this change out.